File: Assignment 1/Program5.c
#include<stdio.h>

void Accept(int iNo)
{
    int iCnt=0;

    for(iCnt=1;iCnt<=iNo;iCnt++)
    {
        printf("*\t");
    }

}

int main()
{
    int iValue=0;

    printf("Enter Number of Stars:");
    scanf("%d",&iValue);

    Accept(iValue);

    return 0;

}

File: Assignment 26/Program5.java
import java.util.*;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt=0,iCount=iNo;

        for(iCnt=1;iCnt<=iNo;iCnt++)
        {            
            System.out.print((iCnt*2)+"\t");
        }

    }
}
class Program5
{
    public static void main(String A[])
    {
        int iValue=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the number:");

        iValue=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue);
    }
}